cli/muxer.c

```diff
@@ -850,14 +850,42 @@ static int open_input_files( muxer_t *muxer )
                 if( !opt->isom && opt->qtff )
                     return ERROR_MSG( "the input seems AV1, at present available only for ISO Base Media file format.\n" );
                 if( opt->isom )
+                {
                     add_brand( opt, ISOM_BRAND_TYPE_AV01 );
+
+                    for( int i = 1; i <= lsmash_count_codec_specific_data( in_track->summary ); i++ )
+                    {
+                        lsmash_codec_specific_t *av1 = lsmash_get_codec_specific_data( in_track->summary, i );
+                        if ( !av1 )
+                            continue;
```

Comment:
Is continuing the right choice here? It seems like for other logic errors the code bails out.

Reply:
In `lsmash_duplicate_summary()` in importer.c, null codec-specific data is ignored, so I think it's best to match that behaviour.

Reply:
ok